students/eunhyekim/users/views.py

```diff
@@ -11,14 +11,21 @@
 class SignInView(View):
     def post(self, request):
         try:
-            data = json.loads(request.body)
-            
-            if not User.objects.filter(email = data["email"], password = data["password"]).exists():
-                return JsonResponse({'message':'INVALID_USER'}, status = 401)
-            return JsonResponse({'message':'SUCCESS'},status = 200)
+            data     = json.loads(request.body)
+            password = User.objects.get(email = data["email"]).password 
+            user_id  = User.objects.get(email = data["email"]).id 
```

Comment:
나머지는 아주 깔끔하게 잘 작성해주셨네요~

line 15, 16에서 `User.objects.get()`를 두번 사용하고 있네요. 그렇게 되면 User 정보를 데이터베이스에 2번 요청하게 됩니다.
아래와 같이 user 변수에 담아서 사용하시면 불필요한 코드와 데이터베이스 요청을 줄일 수 있을 것 같습니다~

```suggestion
            user = User.objects.get(email = data["email"])
```

Reply:
아! 수정하겠습니다 감사합니다!